contributed_definitions/NXem.nxdl.xml

```diff
@@ -21,147 +21,343 @@
+                 
+                 Users should be aware though that even with having both time instances
+                 specified, it may not be possible to infer how long the experiment took or
+                 for how long data were acquired.
+                 
+                 More detailed timing data over the course of the experiment have
+                 to be collected to compute this. These computations can take
+                 advantage of individual time stamps start_time and end_time
+                 in :ref:`NXevent_data_em` instances.
+            </doc>
+        </field>
+        <field name="end_time" type="NX_DATE_TIME" recommended="true">
+            <doc>
+                 ISO 8601 time code with local time zone offset to UTC included when
+                 the microscope session ended. See docstring of the start_time field
+                 to see how the start_time and end_time should be used together.
             </doc>
         </field>
-        <field name="experiment_description" type="NX_CHAR"/>
-        <field name="start_time" type="NX_DATE_TIME"/>
-        <field name="end_time" type="NX_DATE_TIME" recommended="true"/>
         <group type="NXcite" minOccurs="0" maxOccurs="unbounded"/>
-        <group type="NXprogram" minOccurs="1" maxOccurs="unbounded">
-            <field name="program" type="NX_CHAR">
-                <attribute name="version" type="NX_CHAR"/>
-            </field>
-        </group>
-        <!--\@url:-->
         <group type="NXserialized" minOccurs="0" maxOccurs="unbounded">
+            <doc>
+                 Possibility to store a collection of serialized resources that are associated
+                 with the experiment. An example how to use this set could be to document
+                 from which files, which have been e.g. generated by software of technology
+                 partners, the information in an instance of NXem was filled with.
+                 
+                 If resources from technology partners would also be documented with semantic
+                 technology there would not even be a necessity to copy over specific information
+                 into other files as one could write inference and information/fact/knowledge retrieval
+                 algorithm(s) whereby a specific piece of information that is related to an experiment
+                 or simulation is read directly from the respective file of the technology partner.
+                 
+                 The reason why currently this works convincingly in hardly any research data
+                 management system (RDMS) is the strong heterogeneity of the information contained
+                 in such files and the fact that often context and documentation, specifically
+                 rich semantic documentation, and contextualization is missing.
+            </doc>
+            <!--using NXserialized here instead of NXnote as the former is more specific-->
```

Comment:
By September, we shall decide if  we want to propose extending NXnote or adding NXserialized

Reply:
Agreed